Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -499,6 +499,47 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertEqual("Art", segmentClientProperties?["project_category"] as? String)
   }
 
+  // MARK: - Creator Dashboard Properties Tests
+
+  func testCreatorDashboardPageViewed() {
+    let segmentClient = MockTrackingClient()
+    let loggedInUser = User.template |> \.id .~ 23
+    let ksrAnalytics = KSRAnalytics(
+      loggedInUser: loggedInUser,
+      segmentClient: segmentClient
+    )
+
+    ksrAnalytics.trackCreatorDasboardPageViewed()
+
+    XCTAssertEqual(["Page Viewed"], segmentClient.events)
+    XCTAssertEqual(["creator_dashboard"], segmentClient.properties(forKey: "context_page"))
+    XCTAssertEqual(["dashboard"], segmentClient.properties(forKey: "context_section"))
+  }
+
+  func testCreatorDasboardSwitchProjectClicked() {
```

Comment:
So yea just check spelling here and for the test below. If you could also test maybe a specific project property like `project_id` or `name` for the `.template` project that would validate that the right project is being track alongside the event info.